docs/Earn/bridge/bridge-tokens.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Bridge Tokens
+id: bridge-tokens
+---
+
+# Bridge tokens
+
+:::info
+
+Current Bridge version supports bridging aMATICb. Other assets will be added in later releases. 
+
+:::
+
+## Bridge aMATICb
+
+1. Visit [Ankr Bridge](https://www.ankr.com/earn/bridge/). The aMATICb token is chosen by default. 
+2. Enter the desired aMATICb amount to bridge.
+3. Click *Switch network*  to select the correct source and destination networks. Confirm the switch in your wallet.
+4. Click *Approve* to allow Ankr Earn to access your aMATICb and confirm it in your wallet.
+5. Click *Send* to deposit aMATICb to the destination network and confirm it in your wallet.
+   1. We recommend choosing the high fee value for faster transaction mining time.
+6. Ankr Bridge shows you the sending progress. Wait till the cross-chain transaction successfully finishes. 
+   1. Please stay on the page until sending is completed, since next you need to **receive** the bridged tokens. If navigated away, closed the tab, or Ankr Bridge failed during the bridging process, read the **Finish Bridge** section below.
```

Comment:
Consider adding 'Stay on the page until sending has completed in order to receive the bridged tokens. Do not navigate away or close the tab'
As a warning admonition :::caution or :::warning

Reply:
if bridging fails for any reason, see Finish Bridge section (add hyperlink?)

Reply:
thx